spec/controllers/new_administrateur/mail_templates_controller_spec.rb

```diff
@@ -42,4 +42,27 @@
       expect(response.body).to include(procedure.service.telephone)
     end
   end
+
+  describe 'PATCH update' do
+    let(:mail_subject) { 'plop modif' }
+    let(:mail_body) { 'plip modif' }
```

Comment:
Généralement c'est plus facile à comprendre si on met des données plausibles :)

```suggestion
    let(:mail_subject) { 'Mise à jour de votre démarche' }
    let(:mail_body) { '<div>Une mise à jour a été effectuée sur votre démarche n° --demarche-id--.</div>' }
```